ui/app/common/patient-search/models/search.js

```diff
@@ -95,6 +95,12 @@ Bahmni.Common.PatientSearch.Search = function (searchTypes) {
             return patient[column];
         }).join(" - ");
 
+        var abhaIdentifier = null;
+        if (patient.extraIdentifiers) {
+            var objIdentifier = JSON.parse(patient.extraIdentifiers);
+            abhaIdentifier = objIdentifier && objIdentifier["ABHA Address"];
```

Comment:
@angshu - needed some suggestion from you. Can we instead of hardcoding "ABHA Address" in code, make this a configuration field, and then use that `extraIdentifier` to search. So the configuration field could be something like: "searchByExtraIdentfier=<attribute_name>", and then in Clinic_Config india deployment they can set this to `ABHA Address`. This way the code will not mentioned ABHA anywhere (since this is Bahmni Global code, and not india specific).